flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/PredefinedOptions.java

```diff
@@ -191,68 +157,55 @@ public ColumnFamilyOptions createColumnOptions(Collection<AutoCloseable> handles
      *
      * <ul>
      *   <li>setIncreaseParallelism(4)
-     *   <li>setUseFsync(false)
      *   <li>setDisableDataSync(true)
      *   <li>setMaxOpenFiles(-1)
      *   <li>setInfoLogLevel(InfoLogLevel.HEADER_LEVEL)
-     *   <li>setStatsDumpPeriodSec(0)
      * </ul>
      *
      * <p>Note: Because Flink does not rely on RocksDB data on disk for recovery, there is no need
      * to sync data to stable storage.
      */
-    FLASH_SSD_OPTIMIZED {
-
-        @Override
-        public DBOptions createDBOptions(Collection<AutoCloseable> handlesToClose) {
-            return new DBOptions()
-                    .setIncreaseParallelism(4)
-                    .setUseFsync(false)
-                    .setMaxOpenFiles(-1)
-                    .setInfoLogLevel(InfoLogLevel.HEADER_LEVEL)
-                    .setStatsDumpPeriodSec(0);
-        }
+    FLASH_SSD_OPTIMIZED(
+            new HashMap<ConfigOption<?>, Object>() {
+                private static final long serialVersionUID = 462493450763181265L;
```

Comment:
According to the [code guide](https://flink.apache.org/contributing/code-style-and-quality-java.html#java-serialization), we should make the `serialVersionUID` as `1L`. Same for all predefined options.